src/app/_components/profile/UserDetails/UserDetailsDisplay.tsx

```diff
@@ -51,7 +51,7 @@ const UserDetailsDisplay: React.FC = () => {
         </div>
       </CardContent>
       <CardFooter>
-        <DetailsEditDialog />
+        <DetailsEditDialog onDetailsUpdated={refetchUserDetails}/>
```

Comment:
```suggestion
        <DetailsEditDialog refetchUserDetails={refetchUserDetails}/>
```
also change it in the child component if you take the suggestion